OrcScala/src/orc/API.scala

```diff
@@ -48,7 +54,8 @@ trait OrcCompiler[+E] extends OrcCompilerProvides[E] with OrcCompilerRequires
 /** The interface from a caller to an Orc runtime
   */
 trait OrcRuntimeProvides {
-  @throws(classOf[ExecutionException])
+  // TODO: Remove the expression argument on run. It is tied to the token interpreter since other interperters do not use Nameless.
```

Comment:
I'm gonna create an issue to eliminate OrcRuntimeProvides entirely. I think Backend's Runtime totally subsumes it. But it touches so many things I don't want to deal with it now.